ui/src/components/navigation/Sidebar.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FC, ReactElement, useState, useEffect } from 'react';
+import {
+  Box,
+  Drawer,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  useMediaQuery,
+} from '@mui/material';
+import HomeIcon from '@mui/icons-material/Home';
+import MusicNoteIcon from '@mui/icons-material/MusicNote';
+import QueueMusicIcon from '@mui/icons-material/QueueMusic';
+import SupervisedUserCircleIcon from '@mui/icons-material/SupervisedUserCircle';
+import TextSnippetIcon from '@mui/icons-material/TextSnippet';
+
+interface SidebarProps {
+  isOpen: boolean;
+}
+
+const drawerWidth = 240;
+
+const SideBar: FC<SidebarProps> = ({ isOpen }): ReactElement => {
+  const [selectedItem, setSelectedItem] = useState<string | null>(null);
+  const isDesktop = useMediaQuery('(min-width: 769px)');
+  const menuItems = [
+    { icon: <HomeIcon />, text: 'Home' },
+    { icon: <MusicNoteIcon />, text: 'Songs' },
+    { icon: <QueueMusicIcon />, text: 'My Setlists' },
+    { icon: <SupervisedUserCircleIcon />, text: 'Groups' },
+    { icon: <TextSnippetIcon />, text: 'Resources' },
+  ];
+
+  // Reset when switching views
+  useEffect(() => {
+    if (isDesktop) {
+      setSelectedItem(null);
+    }
+  }, [isDesktop]);
+
+  const handleClick = (text: string) => {
+    setSelectedItem(text);
+  };
+
+  return (
+    <Box sx={{ display: 'flex' }}>
+      <Drawer
+        sx={{
+          width: drawerWidth,
+          flexShrink: 0,
+          '& .MuiDrawer-paper': {
+            width: drawerWidth,
+            boxSizing: 'border-box',
+            ...(isDesktop && { position: 'relative' }),
+          },
+        }}
+        // drawer only open in desktop/ clicked on mobile
+        variant={isDesktop ? 'permanent' : 'temporary'}
+        open={isDesktop || isOpen}
+        anchor="left"
+      >
+        <List sx={{ marginTop: { xs: '4em', md: '0' } }}>
+          {menuItems.map((item, index) => (
+            <ListItem
+              key={index}
+              disablePadding
+              sx={{ '&:hover': { backgroundColor: 'primary.light' } }}
+            >
+              <ListItemButton
+                // highlight selected item
+                selected={selectedItem === item.text}
+                onClick={() => handleClick(item.text)}
+              >
+                <ListItemIcon sx={{ color: selectedItem === item.text ? 'primary.main' : '' }}>
```

Comment:
<img width="890" alt="Screenshot 2023-12-02 at 02 17 46" src="https://github.com/hmcc-global/psalted-2/assets/56828805/a7c8efb1-6a3d-45c8-ac5d-f1cb056656b7">
<img width="457" alt="Screenshot 2023-12-02 at 02 17 54" src="https://github.com/hmcc-global/psalted-2/assets/56828805/4c448411-129f-416f-994e-3b380a0152dd">

Not sure if this is a browser problem. In my local, the desktop sidebar is light when selected and in mobile sidebar its primary main. Can u check if its like that in ur local?